src/routes/trip.route.js

```diff
@@ -16,5 +19,6 @@ router.post('/multi-city', authMiddleware.checkUserLoggedIn, authMiddleware.chec
 router.get('/requests', authMiddleware.checkUserLoggedIn, requestValidation, TripController.userTripRequestList); // user request list route
 router.get('/locations', authMiddleware.checkUserLoggedIn, TripController.viewAvailableLocations);
 router.post('/requests/:tripId/comments', authMiddleware.checkUserLoggedIn, UserValidation.validateUserComment, CommentController.addCommentOnTripRequest); // user comment on request trip route
+router.patch('/requests/:tripId/edit-trip-request', authMiddleware.checkUserLoggedIn, editTripRequestValidation, TripController.editOpenTripRequest);
```

Comment:
On this route, I think you should remove `trip-request` for example `/requests/:tripId/edit`

Reply:
Ok did it